.graphqlconfig

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "SkyWalking GraphQL Schema",
+  "extensions": {
+    "endpoints": {
+      "SkyWalking GraphQL Endpoint": {
+        "url": "http://122.112.182.72:8080/graphql",
```

Comment:
I think this is an IP leak.

Reply:
> I think this is an IP leak.

What do you mean by saying "leak", it's a public demo address

Reply:
Everyone who visits the main repo can get access to this address

Reply:
Anyway, fixed